src/lib/utils.js

```diff
@@ -20,3 +39,24 @@ export async function pollUntil( fn, interval, isDone ) {
 		await new Promise( res => setTimeout( res, interval ) );
 	}
 }
+
+/**
+ * Create a temporary directory in the system's temp directory
+ *
+ * @param {string} prefix Prefix for the directory name
+ * @return {string}				Path to the temporary directory
+ * @throws {Error} 				If the directory cannot be created
+ */
+export function makeTempDir( prefix = 'vip-cli' ) {
+	const tempDir = fs.mkdtempSync( path.join( os.tmpdir(), `${ prefix }-` ) );
```

Comment:
Just wanted to follow OS's default system for temporary directory handling. It also gives you a unique name for the directory, so you don't have to worry about conflicts.
We didn't strictly need the manual cleanup, but did it anyway because it doesn't hurt to have it.